<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnifferMainWindow</class>
 <widget class="QMainWindow" name="SnifferMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZigBee Sniffer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/logoZigBee.png</normaloff>:/icons/logoZigBee.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,0">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Recived packets</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeView" name="treePackets">
          <property name="indentation">
           <number>10</number>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Packet info</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeDetails">
          <property name="toolTip">
           <string>Detailed packet info</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">Field</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupFile">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>File</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="btnNew">
           <property name="toolTip">
            <string>New</string>
           </property>
           <property name="text">
            <string>New</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnImport">
           <property name="toolTip">
            <string>Import</string>
           </property>
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnSave">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Save</string>
           </property>
           <property name="text">
            <string>Save as</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupSniff">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Sniff</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="btnSniff">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Start/stop sniffing</string>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnScanChannel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Start/stop channel scan</string>
             </property>
             <property name="text">
              <string>Scan channel</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/page-zoom.png</normaloff>:/icons/page-zoom.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnScanChannels">
             <property name="toolTip">
              <string>Start/stop channels scan</string>
             </property>
             <property name="text">
              <string>Scan channels</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/page-zoom2.png</normaloff>:/icons/page-zoom2.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="treeChannels">
           <property name="toolTip">
            <string>Select channel to operate in</string>
           </property>
           <property name="indentation">
            <number>10</number>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>39</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupNetwork">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Network</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="btnNameNodes">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Name nodes</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnStatistics">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Statistics</string>
           </property>
           <property name="text">
            <string>Statistics</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/view-statistics.png</normaloff>:/icons/view-statistics.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionImport"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSniffer">
    <property name="title">
     <string>Sniffer</string>
    </property>
    <addaction name="actionScan_channels"/>
    <addaction name="actionScan_channel"/>
    <addaction name="actionSniff"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Network</string>
    </property>
    <addaction name="actionStatistics"/>
    <addaction name="actionName_nodes"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSelectPort"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout_sniffer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSniffer"/>
   <addaction name="menuNetwork"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/window-close.png</normaloff>:/icons/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionScan_channels">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/page-zoom2.png</normaloff>:/icons/page-zoom2.png</iconset>
   </property>
   <property name="text">
    <string>Scan channels</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionScan_channel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/page-zoom.png</normaloff>:/icons/page-zoom.png</iconset>
   </property>
   <property name="text">
    <string>Scan channel</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionSniff">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Sniff</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionPort_settings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Port settings</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/view-statistics.png</normaloff>:/icons/view-statistics.png</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionNetwork_structure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/nepomuk.png</normaloff>:/icons/nepomuk.png</iconset>
   </property>
   <property name="text">
    <string>Network structure</string>
   </property>
  </action>
  <action name="actionName_nodes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/mail-tagged.png</normaloff>:/icons/mail-tagged.png</iconset>
   </property>
   <property name="text">
    <string>Name nodes</string>
   </property>
  </action>
  <action name="actionSelectPort">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Select port</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_sniffer">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About sniffer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
